src/pages/greeting/greeting.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import { HomePage } from '../pages/home/home/../';
```

Comment:
please fix bug here

`import { HomePage } from '../pages/home/home/../';`

its not that, change with this : 
every `../` its exit 1 folder to the path location. pls change.

`import { HomePage } from './../../pages/home/home';`